packages/components/src/index.ts

```diff
@@ -1469,7 +1473,6 @@ export type { SelectInputOption, SelectInputProps } from './internal/components/
 export type { PermissionsProviderProps } from './internal/components/permissions/models';
 export type { ISelectInitData } from './internal/components/forms/model';
 export type { QuerySelectOwnProps } from './internal/components/forms/QuerySelect';
-export type { UserProviderProps } from './internal/components/user/UserProvider';
```

Comment:
Was the last external usage of `<UserProvider/>` provider removed as well? Probably best to leave this type exported until we remove the component itself.

Reply:
Doing a github organization search for 'UserProviderProps' only turns up instances where UserProviderProps have been removed, within this branch! In the light of which, it's gonna go.